core/src/main/java/nl/nn/adapterframework/core/Adapter.java

```diff
@@ -60,6 +60,16 @@
 import nl.nn.adapterframework.util.RunStateManager;
 
 /**
+ * An Adapter receives a specific type of messages and processes them. It has {@link Receiver}-s
+ * that determine how messages are received and a {@link PipeLine} that determines how
+ * the messages are processed. If an adapter can receive its messages from multiple sources,
+ * each source appears as a separate {@link Receiver} nested in the adapter. Each {@link Receiver} is also responsible for dealing with
+ * the response of its received messages; the response is the output of the {@link PipeLine}. Each adapter is
+ * part of a {@link Configuration}.
+ *
+ * @author Johan Verrips
+ */
+/*
```

Comment:
Je moet niet zomaar alle oude javadoc naar commentaar veranderen.

Reply:
Ik heb de informatie die er stond in mijn tekst opgenomen. De oude tekst is nu weg.